### ISSUE TITLE
Clarify and clean up usage of SHOULD, MUST, MAY et.c.

### ISSUE THREAD

**Issue by estan** [tavendo/WAMP#166]
I can't find anywhere in the spec where the meaning of SHOULD, MUST, MUST NOT, MAY, MUST BE et.c. is defined. Either it should be defined directly in the spec, or a link to some existing definition should be there.

Also, they're not always in uppercase, so that should probably be looked over.

**Comment by meejah** [comment]
That wording usually comes from https://www.ietf.org/rfc/rfc2119.txt but yes, being explicit would be good.

**Comment by estan** [comment]
Yes, thanks. Sorry was to lazy to dig out the link :) Just a reference to that would be good, and maybe double check the spec for any places where it should be uppercase but isn't, and also e.g. "MUST BE", which I've seen used, should be "MUST be".

**Comment by goeddea** [comment]
The RFC draft now has the link to RFC 2119. Will fix this as I go over the draft.

**Comment by estan** [comment]
:thumbsup:
